packages/x-date-pickers-pro/src/models/index.ts

```diff
@@ -1,3 +1,4 @@
+export type { UseDateRangeFieldProps } from './dateRange';
```

Comment:
Usually I export everything from `src/models` file because if it was private it should be in `src/internals/models`

Do we have something in the `dateRange` file that we don't want to expose?

Reply:
I've reverted the moving of `DayRangeValidationProps` to avoid the need for explicit export. 😉